### DISCUSSION TITLE
nicotine+-3.3.7 on macOS 15.1.1; app won't open

### DISCUSSION THREAD

**Discussion by aesculusglabra** [nicotine-plus/nicotine-plus#3228] [General]
I am following the instructions to 'open anyway' and entering my password. After doing that, nothing happens. The app does not open.

**Comment by mathiascode** [comment]
The 3.3.7 Apple Silicon build is currently broken. The download link has been reverted to 3.3.6.

**Comment by aesculusglabra** [comment]
Awesome – thank you so much!
